test/conformance/kuttl/validate/clusterpolicy/standard/enforce/resource-valid/00-cleanup.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kuttl.dev/v1beta1
```

Comment:
I'm not clear. I thought the bug fix you made was to obviate the need for such cleanup files. Is that not the case?

Reply:
Yes, manual cleanup is not needed anymore.